Src/Plugins/Security/Security.csproj

```diff
@@ -24,13 +24,17 @@
     <Content Include="..\..\..\refs\Octokit.dll">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+	<Content Include="..\..\..\refs\aliyun-net-sdk-Iot.dll">
+		<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
nit: can u align with the others?

Reply:
Done

---
In reply to: [591851295](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591851295) [](ancestors = 591851295)